en/docs/learn/exporting-user-info-to-pardot.md

```diff
@@ -0,0 +1,24 @@
+#Exporting WSO2 Identity Server User Information to Pardot
+
+Pardot is a software as a service (SaaS) marketing automation platform by SalesForce offering email automation,
+targeted email campaigns, and lead management for B2B sales and marketing organizations. This topic provides 
+instructions on exporting user data from WSO2 Identity Server (WSO2 IS) to the Pardot. 
+
+Let's get started!
+
+!!! tip "Before you begin!"
+    Create the `users.csv` file by following the instructions in 
+    [Exporting User Data From WSO2 Identity Server](../learn/exporting-user-data-from-wso2-is.md).
+
+##Try it Out
+
+1. Login to [Pardot](https://pi.pardot.com/).
+2. Navigate to **Admin > Import > Prospects**.
+    ![import-prospect-in-paradot](../assets/img/tutorials/import-prospect-in-paradot.png)
+3. Click **+ Upload File** and choose the created `users.csv` file.
+    ![upload-file-in-paradot](../assets/img/tutorials/upload-file-in-paradot.png)
+4. Click **Next**, after the file get uploaded successfully.
```

Comment:
Once the file gets uploaded successfully, click **Next**.

Reply:
Fixed in fc98995